modules/mono/glue/GodotSharp/GodotSharp/Core/Callable.cs

```diff
@@ -157,7 +157,7 @@ public unsafe void CallDeferred(params Variant[] args)
         /// <list type="number">
         ///    <item>
         ///        <term>delegateObj</term>
-        ///        <description>The given <paramref name="delegate"/>, upcast to <see cref="object"/>.</description>
+        ///        <description>The given <paramref name="delegate"/>, upcast to <see langword="object"/>.</description>
```

Comment:
Ahh, that's fair. Force of habit I suppose, I'm more used to specifying the type explicitly if referencing the type is the intent (eg: `<see cref="System.Object"/>`) & opting for langword otherwise. I'll revert it to the old format in the meantime